themes/scss/infowindow/cartodb-infowindow-default.scss

```diff
@@ -92,8 +96,45 @@
   background: #888 !important;
 }
 
-.is-header .CDB-infowindow-mediaTitle {
+.CDB-infowindow.is-header .CDB-infowindow-mediaTitle {
   bottom: 12px;
+  display: block;
+}
+
+.CDB-infowindow.is-fail .CDB-infowindow-header {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.CDB-infowindow.has-items.has-title .CDB-infowindow-fail {
+  position: absolute;
+  left: 20px;
+  text-align: left;
+}
+
+.CDB-infowindow.has-title .CDB-infowindow-fail {
+  position: absolute;
+  top: 20px;
+  left: 20px;
+  text-align: left;
+}
+
+.CDB-infowindow.has-title .CDB-infowindow-fail {
+  position: absolute;
+  left: 0;
+  top: 20px;
+  width: 100%;
+  text-align: center;
+}
+
+.CDB-infowindow-fail {
+  font-size: 9px;
```

Comment:
Properties should be ordered display, width, color, font-size, text-align, text-transform